app/components/FavoriteInfo.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DISPLAY_NAME } from '../helpers/constants';
+import { calculateStyle } from '../helpers/favUtils';
+
+const FavoriteInfo = ({ images }) => {
+  const preferenceResults = images ? calculateStyle(images) : [];
+
+  return (
+    <div className='page-info'>
+      <h2 className='page-title'>FAVORITES</h2>
+      {images ? preferenceResults.map((result, i) =>
```

Comment:
Clever use of a ternary directly within the return statement. I hadn't thought of that but now I may need to go implement that in a few places.